LiskHQ/lisk
# Invalid undo multisignature unit test - Closes #1527

### What was the problem?
There were two problems:
1. The `undo` transaction root level block was written using `it`, due to which the the nested `it` tests were never executed.
2. Multisignature undo transaction test had wrong expectation against the implementation.
### How did I fix it?
1. Changed the root level block to `describe` due to which the 3 nested `it` tests were skipped are included `(101 tests before and 104 tests now)`.
2. Update multimin, multilifetime expectation to have the same behaviour as per the implementation.
### How to test it?
run `./node_modules/.bin/mocha test/unit/logic/multisignature.js` to see the `undo` transaction being the root level block.
### Review checklist

* The PR solves #1527 
* All new code is covered with unit tests
* All new code was formatted with Prettier
* Linting passes
* Tests pass
* Commit messages follow the [commit guidelines](CONTRIBUTING.md#git-commit-messages)
* Documentation has been added/updated
